src/main.rs

```diff
@@ -29,16 +29,17 @@ use crate::{
     data_generation::QueryGenerator, helpers::run_query, runtime::Runtime, tx_hooks::DemoAppTxHooks,
 };
 mod batch;
+mod context;
 mod data_generation;
 mod helpers;
 mod runtime;
 mod stf;
 mod tx_hooks;
 mod tx_verifier;
 
-type C = MockContext;
+type C = DemoContext;
 type DemoApp = Demo<C, DemoAppTxVerifier<C>, Runtime<C>, DemoAppTxHooks<C>>;
-const CELESTIA_NODE_AUTH_TOKEN: &'static str = "";
+const CELESTIA_NODE_AUTH_TOKEN: &'static str = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJBbGxvdyI6WyJwdWJsaWMiLCJyZWFkIiwid3JpdGUiLCJhZG1pbiJdfQ.nHzh7kWvC3puCYgcMJRuNlMudwf6xGagETNdQyRQQ_s";
```

Comment:
nit: I think you accidentally left this in.

Reply:
Good catch :)